src/CollAction/Controllers/ProjectsController.cs

```diff
@@ -120,9 +126,24 @@ public ViewResult StartInfo()
                 LocationId = createProjectViewModel.LocationId,
                 Target = createProjectViewModel.Target,
                 End = createProjectViewModel.End,
-                Start = DateTime.UtcNow
+                Start = DateTime.UtcNow,
+                BannerImage = null 
             };
-            
+
+            if (createProjectViewModel.HasBannerImageUpload)
+            {
+                var manager = new ImageFileManager
```

Comment:
ImageFileManager probably needs a constructor. It's not a "pure data" class, and there are a couple of ways you could initialize it wrong.

A constructor would ensure people can't initialize the class wrong, and would take a context, a webroot and a webfolder.